src/tests/ftest/launch.py

```diff
@@ -63,6 +63,7 @@
     "ucx+dc_x",
     "ofi+tcp;ofi_rxm",
     "ofi+opx",
+    "ucx+ud_x",
```

Comment:
This list is ordered by preference. Maybe we should move this one to right after the other ucx so it gets chosen over tcp?

Reply:
Okay, I though it was just additions to the list go to the end or something.